src/userPage/index.js

```diff
@@ -287,9 +287,12 @@ function addBeverageButton(beverage) {
 		type: 'button',
 		class: 'btn btn-lg btn-block btn-hover pointer',
 		style: 'margin-top: .5rem'
-	}).text(beverage.name + ' [' + moneyFormat(beverage.price) + ']').click( function(event) {
+	}).text(' (' + beverage.stock + ') ' + beverage.name + ' [' + moneyFormat(beverage.price) + ']').click( function(event) {
```

Comment:
Stock first, then name ?? Maybe we should use a templating engine to determine the button labels.